### ISSUE TITLE
fresh install on chrome complains of "possible conflict"

### ISSUE THREAD

**Issue by nightpool** [hobinjk/XKit#91]
1. create a new chrome profile. (starting chrome with an empty user-data-dir will do this)
2. load the xkit extension
3. load tumblr, wait for xkit to install itself.
4. click "refresh the page to continue using xkit"
5. see that xkit complains of an interfering plugin:

![image](https://cloud.githubusercontent.com/assets/233815/9022749/3c97bb5a-3847-11e5-841a-ec0a8d12a3f5.png)

(this is trivially ludicrous, because there are no other extensions installed)

**Comment by nightpool** [comment]
I had a similar experience testing the new extension on chrome.

**Comment by nightpool** [comment]
the offending line seems to be xkit.js:758
````javascript
    if ($("#dashboard_controls_open_blog").length === 0)
````
which was from the old blog stats bar. Not sure which plugin hides these, but we add them back in. Maybe do the opposite and a check for > 0 would be useful?

**Comment by nightpool** [comment]
also this line is also true:
````js
$(".likes").length === 0 || $(".following").length === 0 || $(".spotlight").length === 0
````
in fact, none of these classes are present at this time

**Comment by dlmarquis** [comment]
@nightpool Tumblr removed that entire div, so that bit of code should be removed. I'll deal with it, but I'm on a roadtrip right now and my internet could cut out at any time, so it might be a couple hours before you see the changes.

**Comment by dlmarquis** [comment]
Just submitted the changes. They'll be incorporated into the next release of XKit.

**Comment by nightpool** [comment]
@dlmarquis I can deal with it, I'm just wondering if there's anything we should put in its place. (we should probably try to detect for old-xkit, for example)

do we have any known conflicts with other tumblr plugins?

**Comment by dlmarquis** [comment]
@nightpool It should be detecting old versions of Xkit already, but that appears to be broken at the moment. It's probably just a matter of bumping the version number.

We know there's a conflict with Stylish that makes XKit unusable, but we don't know why, since Stylish plays nice with the old version. That probably deserves its own issue.

**Comment by nightpool** [comment]
@dlmarquis sounds good.

**Comment by nightpool** [comment]
fixed by #95
